testing/sdk_utils.py

```diff
@@ -166,20 +166,17 @@ def get_metadata() -> requests.Response:
                                    retry=False)
 
 
+@retrying.retry(stop_max_attempt_number=3, wait_fixed=1000, retry_on_result=lambda result: not result)
 def get_cluster_zones():
-    rc, stdout, _ = sdk_cmd.run_cli("node --json")
+    _, stdout, _ = sdk_cmd.run_cli("node --json")
 
-    if rc != 0:
```

Comment:
We should keep
```python
if rc != 0:
    return None
```
here so that the method retries when the `dcos node` command fails. Just not the warning.